modules/packages/Python.chpl

```diff
@@ -2369,13 +2485,17 @@ module Python {
       :arg arr: The Chapel array to create the object from.
     */
     proc init(in interpreter: borrowed Interpreter, ref arr: []) throws
-      where isSupportedArrayType(arr) {
-      super.init(interpreter, ArrayTypes.createArray(arr), isOwned=true);
+    where isSupportedArrayType(arr) {
+      super.init(interpreter, nil: PyObjectPtr, isOwned=true);
       this.eltType = arr.eltType;
+      init this;
+      var g = PyGILState_Ensure();
+      defer PyGILState_Release(g);
+      this.obj = ArrayTypes.createArray(arr);
     }
     @chpldoc.nodoc
     proc init(in interpreter: borrowed Interpreter, ref arr: []) throws
-      where !isSupportedArrayType(arr) {
+    where !isSupportedArrayType(arr) {
```

Comment:
Very, very minor, but I don't like this modification, I think having the where clause indented makes it clearer

Reply:
I tend to disagree, but I don't care enough to worry about it. reverting this